client/modules/_common_components/src/datafiles/DatafilesBreadcrumb/DatafilesBreadcrumb.tsx

```diff
@@ -78,7 +76,10 @@ export const BaseFileListingBreadcrumb: React.FC<
   const { user } = useAuthenticatedUser();
   const rootAlias =
     systemRootAlias || getSystemRootDisplayName(api, system, systemLabel);
-  const systemRoot = isUserHomeSystem(system) ? '/' + user?.username : '';
+  let systemRoot = '';
+  if (system === USER_MYDATA_SYSTEM) systemRoot = '/' + user?.username;
+  if (system === USER_WORK_SYSTEM) systemRoot = '/work/' + user?.homedir;
```

Comment:
This might need to specify that the existing path starts with /work as well, otherwise we get this:
![Screenshot 2024-09-03 at 1 55 32 PM](https://github.com/user-attachments/assets/64fdfafc-153b-4e5b-9773-d3f19332dd6b)